content/How to customize fonts in React Native - Manuela Viola.md

```diff
@@ -0,0 +1,109 @@
+# How to customize fonts in React Native
+
+![Main image](images/customize_fonts_main.png)
+
+## Introduction
+
+Like every newbie in Rootstrap, my first assignment was to develop Target MVD application, in my case, in React Native. While doing this, one of the first things I tried was to set the family font that the application uses, and I found out it wasn't as easy as I expected.
+
+The biggest issue was that all the tutorials said to excecute "react-native link" command and after executing it, the application would stop building successfully.
```

Comment:
The biggest issue was that all the tutorials said to `execute` (watch for the typo here) "react-native link", which in my experience has being really unstable (maybe link a few issues related to `link` you find in the internet?)